tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/FlatJSDependency.scala

```diff
@@ -34,4 +34,6 @@ final class FlatJSDependency(
   @deprecated("Use relPath instead.", "0.6.1")
   val resourceName: String = relPath
 
+  override def toString: String = s"""$origin, dependencies: ${dependencies.mkString(", ")}, commonJSName: $commonJSName, relPathMinified: $relPathMinified"""
```

Comment:
On the content, since there are several things separated by commas, we should surround the whole thing with `FlatJSDependency( ... )`. Otherwise, when you `mkString` a list of `FlatJSDependency, you have no idea where the separations are.

Maybe take inspiration from https://github.com/scala-js/scala-js/blob/47ef7be4d5e05287334b22374cab4beb10085600/tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/JSDependency.scala#L83-L94

The two same comments apply to the other two `toString()`s.